concordium-contracts-common/src/schema_json.rs

```diff
@@ -930,6 +930,41 @@ fn write_bytes_from_json_schema_type<W: Write>(
     }
 }
 
+impl Fields {
+    pub fn field_to_json(&self) -> serde_json::Value {
+        match self {
+            Fields::Named(vec) => {
+                let mut map = Map::new();
+
+                for (name, field) in vec.iter() {
+                    map.insert(name.to_string(), field.schema_type_to_json());
+                }
+
+                serde_json::Value::Object(map)
+            }
+            Fields::Unnamed(vec) => {
+                let mut new_vector: Vec<Value> = vec![];
+                for element in vec.iter() {
+                    new_vector.push(element.schema_type_to_json())
+                }
+
+                serde_json::Value::Array(new_vector)
+            }
+            Fields::None => {
+                let mut map = Map::new();
+                map.insert("None".to_string(), serde_json::Value::Array(Vec::new()));
+                serde_json::Value::Object(map)
```

Comment:
This should just be an empty vector. 
So that it will output:
`"MyEnumVariant" : []`

See https://github.com/Concordium/concordium-client/blob/30db64c1e3f4ba8d05c41080e52af47bcef909e3/src/Concordium/Client/Types/Contract/Schema.hs#L424